production/helm/loki/templates/_helpers.tpl

```diff
@@ -80,7 +80,7 @@ Common labels
 helm.sh/chart: {{ include "loki.chart" . }}
 {{ include "loki.selectorLabels" . }}
 {{- if or (.Chart.AppVersion) (.Values.loki.image.tag) }}
-app.kubernetes.io/version: {{ .Values.loki.image.tag | default .Chart.AppVersion | quote }}
+app.kubernetes.io/version: {{ (regexReplaceAllLiteral "@.*" (.Values.loki.image.tag | default .Chart.AppVersion) "") | quote }}
```

Comment:
2 questions.

1. should we replace it with a `:` instead of nothing so there's still a visible separator
2. we probably also want to enforce it's truncated to 63 characters?

Reply:
Thanks for your review @trevorwhitney.

As `:` is not allowed, I've replaced it with `-`. I've moved this to a new defined template `"loki.validLabelValue"`. Back to you :ping_pong:!